src/data/faq.json

```
@@ -173,6 +173,18 @@
                             "Using this step-by-step guide, you can remove certificates from the Corona-Warn-App. Please note that the first three steps apply to both iOS and Android, but only one of the last two steps needs to be followed, depending on which operating system matches yours. Vaccination certificates, test certificates and recovery certificates are mentioned as 'Certificates', since the process of removing them is identical.",
                             "<ul><li>Tap on the 'Certificates' tab</li><li>Tap on the certificate you would like to remove</li><li>Scroll down to and tap on the tile that matches the certificate you want to delete.</li><li>iOS: Tap on 'Delete Certificate' at the bottom of the screen</li><li>Android: Tap on the three-dot symbol on the top right of the screen, and then tap on 'Remove'</li></ul>"
                         ]
+                    },
+                    {
+                        "title": "Why do I get the error message that my certificate is invalid?",
+                        "anchor": "hc_signature_invalid",
+                        "active": true,
+                        "textblock": [
+                            "The signature of your certificate is invalid.",
+                            "This invalidates the entire certificate as a digital document. Therefore, it cannot be used as a proof and also cannot be imported into the app. If this error message occurs with a certificate that has already been imported, it is now marked as invalid.",

```

```suggestion
                            "This invalidates the entire certificate as a digital document. Therefore, it cannot be used as proof of health status and also cannot be imported into the app. If this error message occurs with a certificate that has already been imported, it is now marked as invalid.",
```

In this context "proof" is an uncountable noun, so "a proof" is not correct - see https://www.macmillandictionary.com/dictionary/british/proof_1.  I suggest instead "proof of health status".

The EU page https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_en disagrees on this point where it writes "An EU Digital COVID Certificate is a digital proof that a person has either been vaccinated against COVID-19, received a negative test result or recovered from COVID-19.", I would still favor my suggestion above though.

---

Changed it, thanks!